client/src/components/Games/GamesList.scss

```diff
@@ -1,6 +1,6 @@
 .pagination{
     display: inline-block;
-    margin-right: 5px;
+    // margin-right: 5px;

```

Comment:
Don't worry. Be happy, just delete it. We have git for a reason.

Reply:
Deleted 👍